### DISCUSSION TITLE
Can I access the value in an env file from a python script using os.getenv?

### DISCUSSION THREAD

**Discussion by Siedlerchr** [community/community#25415] [Actions]
<p>Hi,</p>
<p>I have a question about the new environment files we should use instead of the set env. I am using an env variable to store the current tag number to pass it to a python script.</p>
<pre><code class="lang-auto">      - name: Setup Env Variable only on tag
        run: echo ::set-env name=GH_BUILD_NR::${GITHUB_REF/refs\/tags\//}
        if: startsWith(github.ref, 'refs/tags/')
</code></pre>
<p>And in my python script I used:</p>
<pre><code class="lang-auto">build_number = os.getenv('GH_BUILD_NR')
</code></pre>
<p>Does this still work with the new files?</p>

**Comment by airtower-luna** [comment]
<p>Yes, under the same conditions as before: The Python script must be called in a later step that’s part of the same job.</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/siedlerchr">@siedlerchr</a>,</p>
<p>Sure, using “<strong>os.getenv(‘ENV_NAME’)</strong>” or “<strong>os.environ[‘ENV_NAME’]</strong>” in Python scripts to access environment variables still can work.</p>
<p>In addition, the old workflow command (<strong>echo “::set-env name={name}::{value}”</strong>) to set an environment variable has been deprecated.<br>
Currently, the old command still can work in the workflows, however we recommend that you use the new command (<strong>echo “{name}={value}” &gt;&gt; $GITHUB_ENV</strong>).<br>
More details, see “<a href="https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable" rel="noopener nofollow ugc"><strong>Setting an environment variable</strong></a>”.</p>
